model/task/task.go

```diff
@@ -1313,6 +1313,23 @@ func FindMidwayTask(t1, t2 Task) (*Task, error) {
 	return FindOne(db.Query(ByRevisionOrderNumber(t1.BuildVariant, t1.DisplayName, t1.Project, t1.Requester, mid)))
 }
 
+// FindMidwayTaskFromIds gets the task between two task given that they are
```

Comment:
nit: two tasks. I also think its implied that the order of ids does not matter so let's remove that and just start the next sentence with `If the task passed cannot have...`